src/path.js

```diff
@@ -0,0 +1,45 @@
+class Path {
+    constructor(nodes, edges){
+        this.nodes = nodes;
+        this.edges = edges;
+    }
+
+    getNodes(){
+        return this.nodes;
+    }
+
+    getEdges(){
+        return this.edges;
+    }
+
+    getNode(index){
```

Comment:
should you use gettter/setter?

Reply:
please elaborate

Reply:
it is a property getter, so done, for every zero parametes function